src/Model/Contact.php

```diff
@@ -3106,6 +3106,13 @@ public static function createFromProbeForUser(int $uid, string $url, string $net
 			$ret['url'] = str_replace('?absolute=true', '', $ret['url']);
 		}
 
+		if (($protocol == Protocol::ACTIVITYPUB) && ($uid != 0)) {
+			if (APContact::isRelay(APContact::getByURL($ret['url']))) {
```

Comment:
These conditions could have been combined together. I assume they were part of a more elaborate condition structure?

Reply:
They were part of a coding at 6 AM in a half-awake state 😴